packages/@expressive-code/plugin-shiki/test/rendering.test.ts

```diff
@@ -25,6 +25,20 @@ npm create astro@latest -- --template <example-name>
 npm create astro@latest -- --template <github-username>/<github-repo>
 `.trim()
 
+const ansiTestCode = `
+[95mRunning tests from 'C:\\Git\\Pester\\expressiveCodeAnsiDemo.tests.ps1'[0m
+[92mDescribing Expressive Code[0m
+[96m Context Testing ANSI[0m
+[32m   [+] Success[0m[90m 3ms (1ms|2ms)[0m
+[91m   [-] Failure[0m[90m 2ms (2ms|0ms)[0m
+[91m    Expected 2, but got 1.[0m
+[97mTests completed in 47ms[0m
+[97mTests Passed: 1 [0m[91mFailed: 1 [0m[90mSkipped: 0 [0m[37m[0m[90mNotRun: 0[0m
+`.trim()
+
+// eslint-disable-next-line no-control-regex
+const ansiEscapeCode = /\u001b\[\d+m/gmu
```

Comment:
```suggestion
const ansiEscapeCode = /\u001b\[\d+m/gu
```

You're right, I think we can at least remove the multiline flag that doesn't seem to be needed here. I'd leave the global flag in case we want to match multiple escape sequences in future tests.